packages/vscode-extension-pmml-editor/CHANGELOG.md

```diff
@@ -0,0 +1,6 @@
+# 0.8.3 (alpha)
```

Comment:
Is the PMML editor extension going to be released on `0.8.4`?

Reply:
Hi @caponetto possibly.. but I get what you're on about.

I'll change this to 0.8.4 BUT IDK when that's planned and whether we would have published the editor to `npm` by then.

This PR will remain draft until we've had a release of the editor to `npm` and then this will reflect the released version!